save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/SandboxView.kt

```diff
@@ -196,30 +213,9 @@ class SandboxView : AbstractView<SandboxViewProps, SandboxViewState>(true) {
     private fun ChildrenBuilder.renderCodeEditor() {
         div {
             className = ClassName("")
-            codeEditorComponent {
+            sandboxCodeEditorComponent {
                 editorTitle = ""
-                selectedFile = state.selectedFile
-                onSelectedFileUpdate = {
-                    setState { selectedFile = it }
-                }
-                editorText = when (state.selectedFile) {
-                    FileType.CODE -> state.codeText
-                    FileType.SAVE_TOML -> state.configText
-                    FileType.SETUP_SH -> state.setupShText
-                    else -> "Press on any of the buttons above to start editing"
-                }
-                onEditorTextUpdate = {
-                    setState {
-                        when (state.selectedFile) {
-                            FileType.CODE -> codeText = it
-                            FileType.SAVE_TOML -> configText = it
-                            FileType.SETUP_SH -> setupShText = it
-                            else -> {}
-                        }
-                    }
-                }
-                doUploadChanges = ::uploadChanges
-                doReloadChanges = ::reloadChanges
+                currentUserInfo = props.currentUserInfo ?: UserInfo("")
```

Comment:
May be create some extension method for such users `UserInfo("")`? Somewhere we use `UserInfo("Unknown")`, somewhere `UserInfo("Undefined")` and so on. May be we can unify them into `UserInfo("Guest")` or something like this?

Reply:
Well maybe, but talking about this PR it will be removed when #1310 will be merged

But adding something like 
```Kotlin
UserInfo?.orDefault(): UserInfo
```
might be nice